blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/AstyanaxReader.java

```diff
@@ -398,7 +398,7 @@ public MetricData getHistogramsForRange(Locator locator, Range range, Granularit
 
         ColumnFamily cf = CassandraModel.getColumnFamily(HistogramRollup.class, granularity);
         Points<HistogramRollup> histogramRollupPoints = getDataToRoll(HistogramRollup.class, locator, range, cf);
-        return new MetricData(histogramRollupPoints, getUnitString(locator), MetricData.Type.HISTOGRAM);
+        return new MetricData(histogramRollupPoints, null, MetricData.Type.HISTOGRAM);
```

Comment:
@tilogaat We should refactor `MetricData` to stop taking `unit` as a field, so that we can remove it from its constructor as well.